src/datumaro/plugins/transforms.py

```diff
@@ -1126,39 +1164,43 @@ def transform_item(self, item):
         if item.media.has_data:
             resized_image = self._lazy_resize_image(item.media, new_size)
 
-        resized_annotations = []
-        for ann in item.annotations:
-            if isinstance(ann, Bbox):
-                resized_annotations.append(
-                    ann.wrap(
-                        x=ann.x * xscale,
-                        y=ann.y * yscale,
-                        w=ann.w * xscale,
-                        h=ann.h * yscale,
+        def resize_annotations():
```

Comment:
How about moving this out into a class method? Should make the function a bit easier to read.